harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -1010,7 +1010,7 @@ def build_callbacks(self) -> Dict[str, pytorch.PyTorchCallback]:
         """
         return {}
 
-    def evaluate_batch(self, batch: pytorch.TorchData, batch_idx: int) -> Dict[str, Any]:
+    def evaluate_batch(self, batch: pytorch.TorchData) -> Dict[str, Any]:
```

Comment:
This is a public API, please don't make causal changes to it as part of other changes.

If there is a change you want to make to it, you probably should propose it in a public channel first.

Reply:
You're absolutely right. This was an erroenous change I made while experimenting. I've reverted it already.